source/manual/icinga.html.md

```diff
@@ -0,0 +1,66 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Icinga
+section: Monitoring
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-08-16
+review_in: 6 months
+---
+
+- Production
+  - [AWS](https://alert.blue.production.govuk.digital)
+  - [Carrenza](https://alert.publishing.service.gov.uk)
+- Staging
+  - [AWS](https://alert.blue.staging.govuk.digital)
+  - [Carrenza](https://alert.staging.publishing.service.gov.uk)
+- Integration
+  - [Carrenza](https://alert.integration.publishing.service.gov.uk)
```

Comment:
```suggestion
  - [AWS](https://alert.integration.publishing.service.gov.uk)
```

Reply:
Since Integration and CI only exist in AWS and Carrenza (respectively) I'm not sure we need them as a specific subpoint, but that's just my opinion.

Reply:
Would you prefer it as `- [Integration (AWS)](https://...)` or something else? I think the list might end up looking messier.

Reply:
I was thinking just `[Integration](http://...)` since we only have it on one cloud provider.

Reply:
It's quite nice to know which cloud provider it's running on though - especially if the domain name is inconsistent between environments on the same cloud provider.

Reply:
Yeah, I don't really mind either way. Personally I think this page https://docs.publishing.service.gov.uk/manual/govuk-in-aws.html covers that enough that we don't need here, but up to you.